js/app.js

```diff
@@ -0,0 +1,93 @@
+const addForm = document.querySelector('.add');
+const list = document.querySelector('.books');
+const search = document.querySelector('.search input');
+const newBook = document.querySelector('.newbook');
+const hideForm = document.querySelector('.hideform');
+
+const books = [
+  {
+    title: 'The Lord of the Rings', author: 'J. R. R. Tolkien', pages: 1197, read: false, id: 1,
+  },
+];
+
+function Book(title, author, pages, read, id) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+  this.id = id;
+}
+
+const addBookToLibrary = () => {
+  list.innerHTML = '<div></div>';
+  books.forEach((book) => {
+    list.innerHTML += `
+      <div class="list-group-item">
+          <div class="d-flex w-100 justify-content-between">
+          <h5 class="mb-1 title"><small>Title: </small><br>${book.title}</h5>
+          <small class="pages">#pages: (${book.pages})</small>
+          </div>
+          <p class="mb-1 author"><small>Author: </small><br>${book.author}</p>
+          <small class="status" onCLick="toggle(${book.id})">${book.read ? 'Read' : 'To Read'}</small>
+          <small class="float-end remove">Remove</small>
+      </div>
+      `;
+  });
+};
```

Comment:
[OPTIONAL] It will be nice if  `addBookToLibrary` function will take and store the new Book objects into an array not to display books. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library))